microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -109,9 +111,12 @@ The following are some of the flags we've seen used.
 | `msOverlayScrollbarWinStyle` | Whether the users can change between overlay and non-overlay modes for Fluent scrollbars. |
 | `msPdfEnableAsPreview` | This features enables the PDF viewer to launch with a minimal toolbar and in read-only preview mode. |
 | `msSmartScreenProtection` | If enabled, SmartScreen protection will be available. |
+| `msUseSpellCheckCorrectionsCard` | If enabled, a new corrections card UI is shown when the user clicks on the error range of a misspelling or grammar error. |
```

Comment:
```suggestion
| `msUseSpellCheckCorrectionsCard` | If enabled, a new corrections card UI is shown when the user clicks in a range of code that includes a misspelling or grammar error. |
```
@monica-ch - review this Suggested Change.  trying to clarify text in this cell.